yt_dlp/extractor/floatplane.py

```diff
@@ -98,12 +90,12 @@ def format_path(params):
                             hls_aes['key'] = urlh.read().hex()
 
                 formats.append({
-                    **traverse_obj(quality, {
+                    **traverse_obj(variant, {
                         'format_note': ('label', {str}),
-                        'width': ('width', {int}),
-                        'height': ('height', {int}),
+                        'width': ('meta', 'video', 'width', {int}),
+                        'height': ('meta', 'video', 'height', {int}),
```

Comment:
Instead, as they provide much more info now, we can do the following:
```suggestion
                        'width': ('meta', 'video', 'width', {int_or_none}),
                        'height': ('meta', 'video', 'height', {int_or_none}),
                        'vcodec': ('meta', 'video', 'codec', {str}),
                        'acodec': ('meta', 'audio', 'codec', {str}),
                        'vbr': ('meta', 'video', 'bitrate', 'average', {int_or_none(scale=1000)}),
                        'abr': ('meta', 'audio', 'bitrate', 'average', {int_or_none(scale=1000)}),
                        'audio_channels': ('meta', 'audio', 'channelCount', {int_or_none}),
                        'fps': ('meta', 'video', 'fps', {float_or_none}),
```